### ISSUE TITLE
SOTA graphs 2022 (list in progress need to refine / select)

### ISSUE THREAD

**Issue by thegodone** [aimat-lab/gcnn_keras#58]
- [] - MAT
- [] - MolNet
- [] - RGAT
- [] - RGIN
- [] - E(n) Equivariant GNN https://arxiv.org/abs/2102.09844  &  https://arxiv.org/abs/2102.03150 lucidrains/egnn-pytorch  vgsatorras/egnn
- [] - Equivariant GNN for 3D Macromolecular Structure 
- [] - SE(3)-equivariant GNN https://arxiv.org/abs/2106.02347
- [] - Steerable-E3-GNN
- [] - GemNet: Universal Directional Graph Neural Networks for Molecules TUM-DAML/gemnet_tf

**Comment by PatReis** [comment]
At the moment we worked on crystal graph models. We added CGCNN.
I want to add E3GNN or ALIGNN next.

**Comment by thegodone** [comment]
I think E3GNN is a nice addition yes

**Comment by thegodone** [comment]
Would tou have Time to look at rgin or cdmpnn ?

**Comment by thegodone** [comment]
thanks for the CMPNN code. I have another suggestion => https://github.com/chao1224/n_gram_graph

**Comment by thegodone** [comment]
In the paper CMPNN they said Max pooling and in the code you put softmax is there any reason for that change ?

**Comment by PatReis** [comment]
Yeah, sorry I should have put "segment_max". I mixed it up, which is why the error you sent me occured. In attention you have softmax and multiply with edges and then pool via sum. But here it seems that is sumed and afterwards the nodes multiplied with max. I would have to check the tensors to see why the have a [0] here...
I run training again...

**Comment by thegodone** [comment]
I just add this paper that you should look at https://pubs.acs.org/doi/10.1021/acs.jctc.1c01021

**Comment by thegodone** [comment]
GemNet is very existing model

**Comment by thegodone** [comment]
I just add this paper that I forget : https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9189862
it's a variant of GIN that is very close to AttFP / Chemprop in performances
![image](https://user-images.githubusercontent.com/1186658/187597020-599764d3-1275-4b57-9f91-4d79949cfbd6.png)

**Comment by thegodone** [comment]
I just Add the RMAT that look like a rbf + envelop improvement over MAT logic

**Comment by thegodone** [comment]
I encourage to do this https://github.com/HannesStark/3DInfomax

**Comment by thegodone** [comment]
https://arxiv.org/pdf/2111.06283.pdf https://github.com/KarolisMart/DropGNN looks very interesting for MPNN / DMPNN / GIN

**Comment by thegodone** [comment]
this look cool : https://github.com/invokerqwer/Chemical-AI

**Comment by thegodone** [comment]
Reading https://arxiv.org/pdf/2003.00982.pdf I suggest to add the PE versions see AQSOL performances Table 4 (GatedGCN-E-PE).
